x-pack/legacy/plugins/maps/public/layers/layer_wizard_registry.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+/* eslint-disable @typescript-eslint/consistent-type-definitions */
+
+type LayerWizard = {
+  description: string;
+  icon: string;
+  id: string;
```

Comment:
I don't think `id` should be necessary. If so, I also would not re-use the source-type in most instances.

Why not have the layerWizard itself can generate the ID (?)